### ISSUE TITLE
Rework `.toPython()` to use common `StringBuilder`, not plain `String`s

### ISSUE THREAD

**Issue by krzema12** [krzema12/kotlin-python#72]
Originally created by @SerVB.

**Comment by SerVB** [comment]
I already tried that some time ago but AFAIR it didn't give much boost: https://github.com/krzema12/kotlin-python/tree/2021-07-30-stringbuilder

**Comment by krzema12** [comment]
Yeah, I guess we concluded that the current code looks and works good enough. Feel free to close this one if you want.

**Comment by SerVB** [comment]
Let's leave it open until one more try in the far future
